src/web/components/elements/TextBlockComponent.tsx

```diff
@@ -108,9 +108,23 @@ export const TextBlockComponent: React.FC<Props> = ({
     forceDropCap,
     isFirstParagraph,
 }: Props) => {
-    const { willUnwrap: isUnwrapped, unwrappedHtml } = unwrapHtml({
-        prefix: '<p>',
-        suffix: '</p>',
+    const {
+        willUnwrap: isUnwrapped,
+        unwrappedHtml,
+        unwrappedElement,
+    } = unwrapHtml({
+        fixes: [
+            {
+                unwrappedElement: 'p',
+                prefix: '<p>',
+                suffix: '</p>',
+            },
+            {
+                unwrappedElement: 'ul',
+                prefix: '<ul>',
+                suffix: '</ul>',
+            },
```

Comment:
This is the main part that fixes the ul issue - because we can now check multiple elements, we can ensure that we're unwrapping `ul`s too.

Reply:
+1